lib/version-check.js

```diff
@@ -41,7 +41,7 @@ module.exports = function (gulp, gutil) {
 
                 //On success invoke callback function
                 if (typeof cb !== 'undefined') {
-                    cb();
+                    return cb();
```

Comment:
could be "version check passed" or something similar.  Your change above is find for now.